variants/frontend-base/template.rb

```diff
@@ -37,26 +39,13 @@
 empty_directory_with_keep_file "app/frontend/images"
 copy_file "app/frontend/stylesheets/application.scss"
 copy_file "app/frontend/stylesheets/_elements.scss"
-append_to_file "app/frontend/packs/application.js" do
+prepend_to_file "app/frontend/packs/application.js" do
   <<~EO_CONTENT
 
   import '../stylesheets/application.scss';
   EO_CONTENT
 end
 
-enable_imgs_src = <<~EO_SRC
-  //
-  // const images = require.context('../images', true)
-  // const imagePath = (name) => images(name, true)
-EO_SRC
-enable_imgs_replacement = <<~EO_REPLACEMENT
-
-  const images = require.context('../images', true);
-  // eslint-disable-next-line no-unused-vars
-  const imagePath = name => images(name, true);
```

Comment:
Not sure what to replace here!

The current replacement doesn't work anymore, complains with the changes. Removing this whole section seems not giving any complaints.

Reply:
I think if things are working without it, then it's fine to remove it - ultimately since this is a template repo, if we find a common pattern is broken later we can do a PR implementing support for it.